### PR TITLE
Diana's Cow

### PR DESCRIPTION
**Pull request by D-K0**
added Cow class and completed Todo on Trader

### CODE REVIEW

File: src/test/java/CowTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    @Before
+    public void setUp() throws Exception {
+        c = new Cow();
+    }
+
+//    @Test(timeout = 50)
+//    public void TestSound() {
+//        assertEquals("Moo!", c.sound());
+//    }
+
```
Comment (mchan7841) [inline src/test/java/CowTest.java:23]: Missing test for TestSound() (Commented Out)
)

File: src/main/java/Cow.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
+public class Cow implements Drivable, Domesticatable, Tradable {
```
Comment (mchan7841) [inline src/main/java/Cow.java:3]: Missing documentation throughout code.

File: src/test/java/CowTest.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+public class CowTest {
+    Cow c;
+
+    @Before
+    public void setUp() throws Exception {
+        c = new Cow();
+    }
+
+//    @Test(timeout = 50)
```
Comment (Skaurm) [inline src/test/java/CowTest.java]: Commented out test case

File: src/main/java/Cow.java (hunk 2 above)
Comment (Skaurm) [inline src/main/java/Cow.java:3]: Missing documentation

File: src/test/java/CowTest.java (hunk 3 above)
Comment (shayaanmk) [inline src/test/java/CowTest.java]: Add test case for TestSound()

**Review by shayaanmk — CHANGES_REQUESTED** [review]
Need to add test case.

**Review by shayaanmk — APPROVED** [review]
Good to merge

**Pull request merged**